mock/client.go

```diff
@@ -0,0 +1,21 @@
+package mock
+
+import (
+	"net/http"
+
+	recurly "github.com/blacklightcms/go-recurly"
+)
+
+// NewClient sets the unexported fields on the struct and returns a Client.
+func NewClient(httpClient *http.Client) *recurly.Client {
+	if httpClient == nil {
+		httpClient = http.DefaultClient
+	}
+
+	client := recurly.NewClient("a", "b", httpClient)
+
+	client.Subscriptions = &SubscriptionsService{}
+	client.Transactions = &TransactionsService{}
+
```

Comment:
Added in [5238e2](https://github.com/blacklightcms/go-recurly/pull/10/commits/322b61900754b8bcf3719fcf55a8b1a7955238e2)